features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionMetricDispatcher.kt

```diff
@@ -50,4 +50,9 @@ internal interface SessionMetricDispatcher {
      * Called when a missed event is tracked by this session metric.
      */
     fun onMissedEventTracked(sessionId: String, missedEventType: SessionEndedMetric.MissedEventType)
+
+    /**
+     * Called when skipped frame is tracked by this session metric.
+     */
+    fun onSkippedFrameTracked(sessionId: String)
```

Comment:
> [!NOTE]
> Nitpicking: maybe we want to mention that this is regarding SessionReplay, and not another kind of skipped frame (e.g. in the rendering pipeline, compose or other vitals we could track). 
#NamingIsHard™️

Reply:
I changed the name to `onSessionReplaySkippedFrameTracked` for all the related functions